imagebuildah/chroot_symlink.go

```diff
@@ -19,6 +19,10 @@ const (
 	maxSymlinksResolved    = 40
 )
 
+var (
+	errDanglingSymlink = errors.New("error evaluating dangling symlink")
```

Comment:
Creating this with the `github.com/pkg/errors` package adds a traceback that points to this line whenever we return the error.  Can you change this to be created with the standard library's `errors` package, either by importing it under a different name or making it the sole definition in an `errors.go` that uses the standard `errors` package?

Reply:
Fixed. Created a new file called errors.go and used the standard library's error package.